ci-operator/step-registry/openshift-qe/installer/bm/day2/perf-profile/rds-core/openshift-qe-installer-bm-day2-perf-profile-rds-core-commands.sh

```diff
@@ -62,6 +66,7 @@ EOF
 
 sleep 60
 kubectl wait --for jsonpath='{.status.updatedMachineCount}'="$(oc get node --no-headers -l node-role.kubernetes.io/worker= | wc -l)" --timeout=30m mcp worker
+oc adm wait-for-stable-cluster --minimum-stable-period=2m --timeout=20m
```

Comment:
Not a blocking stuff, but maybe this command should better go here instead? https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/installer/bm/day2/perf-profile/rds-core/openshift-qe-installer-bm-day2-perf-profile-rds-core-commands.sh#L65